ffi/pic.lua

```diff
@@ -372,4 +372,84 @@ function Pic.openDocument(filename)
     end
 end
 
+--[[
+function for writing BMP to file
+]]
+
+local function write_uint32(of, data )
+    of:write( string.char(data % 256) )
```

Comment:
An `& 255` (well, `bit.and(data, 255)` in Lua ;p)  will be faster (and identical, because unsigned maths and 256 is a power of two).